docs/management/api/list-endpoints-api.asciidoc

```diff
@@ -1,11 +1,313 @@
 [[list-endpoints-api]]
-=== List endpoints
+=== List Endpoints
 
-Enter API description here.
+Retrieves a list of hosts running {endpoint-sec}.
 
-NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Console].
+NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Run Elasticsearch API requests].
 
 ==== Request URL
 
+`POST <kibana host>:<port>/api/endpoint/metadata`
+
 
 ==== Request body
+
+a JSON object with these fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`paging_properties` |Array a|A set of page instruction Objects that contain the following:
+
+* `page_size` (number): size of results per page. 1 to 10000
+* `page_index` (number): the zero-based page of results to retrieve
+
+A sample value for `paging_properties` would be:
+
+`paging_properties: [{page_size: 20}, {page_index: 0}]`
+
+ |No
+|`filters` |Object a|Filters to apply to the list query. It may contain the following filters as properties:
+
+* `kql` (string): a KQL string
+* `host_status` (array, string): a set of agent health statuses to filter by.
+
+Accepted health status values are:
+
+* `healthy`
+* `unhealthy`
+* `offline`
+* `updating`
+* `inactive`
+* `unenrolled`
+
+|No
+|==============================================
+
+
+===== Example requests
+
+List Endpoints
+
+[source,sh]
+--------------------------------------------------
+POST /api/endpoint/metadata
+{
+	"paging_properties": [
+		{ "page_size": 10 },
+		{ "page_index": 0 }
+	]
+}
+--------------------------------------------------
+// KIBANA
+
+List Endpoints, filtered by
+
+[source,sh]
+--------------------------------------------------
+POST /api/endpoint/metadata
+{
+	"paging_properties": [
+		{ "page_size": 10 },
+		{ "page_index": 0 }
+	]
+}
```

Comment:
@pzl this API call is the same is `List Endpoints`. Doesn't appear to have any of the filter properties. Is this correct?

Reply:
Correct, this is List without filters. We should probably add another example with the filters used